src/main/java/com/bart/scorebetlive442/entity/MatchEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.bart.scorebetlive442.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Entity
+@Table(name = "Match")
+public class MatchEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false, updatable = false)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "team_home_id", nullable = false)
+    private TeamEntity teamHome;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "team_away_id", nullable = false)
+    private TeamEntity teamAway;
+
+    @Column(nullable = false)
+    private Date dateTime;
+
+    @Column(nullable = false)
+    private String stadiumName;
```

Comment:
to jest ciekawy przykład normalizacji baz danych:

teoretycznie to pole jest zbędne, StadiumName, wyciągniemy (albo powinno należeć do encji Team, to drużyna ma stadion - być może to powinna być encja 1-1 / 1-* (stadion jest własnością drużyny, tj. person ma surname). więc informację na jakim stadionie rozegrany był mecz wyciągniemy po Match.TeamHome.Stadium - no ale możemy rozważyć sytuację że jest finał LM gdzie mamy teoretycznie gospodarza ale gra się na innym stadionie wtedy to pole może zostać (wtedy powinniśmy rozważyć stadion jako encję)

Reply:
możemy to omówić na lekcji?